### ISSUE TITLE
Accessibility-> The focus when navigating using the keyboard is not clear and missing

### ISSUE THREAD

**Issue by kgorana** [bcgov/bcparks-ar-api#266]
### Description
**How to replicate:**
* Navigate to the A&R website on the test environment: [A&R Admin](https://d3ahrzu0mpndj3.cloudfront.net/)
* Navigate through the website using the tab key

**Expected Behavior:**
* When navigating using the keyboard, the keyboard focus indicator should be visible

**Issue:**
* The keyboard focus is missing and not clear in most of the places on this website and should be revisited in general
  eg: Try navigating through the Enter data page

  ![Image](https://github.com/bcgov/bcparks-ar-api/assets/145403218/13923f70-abfa-49d8-a3e1-3925ecc54455)

## Linked issues
relates to
- [x] ~[BRS-822](https://bcparksdigital.atlassian.net/browse/BRS-822)~
